graphics/MoltenVK/Portfile

```diff
@@ -53,27 +45,75 @@ destroot {
         file copy ${output_dir}/MoltenVK/MoltenVK.xcframework ${destroot}${frameworks_dir}
     }
 
-    file copy ${output_dir}/macOS/bin/MoltenVKShaderConverter ${destroot}${prefix}/bin
-    file attributes ${destroot}${prefix}/bin/MoltenVKShaderConverter -permissions +x
-    file copy ${output_dir}/macOS/lib/libMoltenVK.dylib ${destroot}${prefix}/lib
+    file copy ${output_dir}/MoltenVK/dylib/macOS/libMoltenVK.dylib ${destroot}${prefix}/lib
 
     # vulkan and vk_video are provided via vulkan-headers
     file copy ${output_dir}/MoltenVK/include/MoltenVK ${destroot}${prefix}/include
 
-    if {(![variant_isset universal])} {
+    if ![variant_isset universal] {
         system -W ${destroot}${prefix}/lib "lipo -thin ${configure.build_arch} libMoltenVK.dylib -o libMoltenVK.dylib 2> /dev/null"
     }
 }
 
-platform darwin {
-    if {${os.major} < 16} {
-        archive_sites
-        distfiles
-        depends_build
-        known_fail  yes
-        pre-fetch {
-            ui_error "${subport} @${version} requires macOS Sierra or later"
-            return -code error "incompatible OS X version"
+if {${subport} eq ${name}} {
+    PortGroup           stub 1.0
+    version             1.0
+    epoch               2
+    if {${os.major} < 17} {
+        depends_run     port:MoltenVK-1.1.9
+    } else {
+        depends_run     port:MoltenVK-latest
+    }
+}
+
+# GitHub artifacts deployment target is 10.13 (Xcode14+)
+subport MoltenVK-latest {
+    platforms           {macosx any >= 17}
+    conflicts           MoltenVK-1.1.9
+}
+
+subport MoltenVK-1.1.9 {
+    github.setup        KhronosGroup MoltenVK 1.1.9 v
+    epoch               1
+    revision            1
```

Comment:
This is a new subport, so the revision (as well as epoch as mentioned above) can start at 0.

Reply:
Agreed this doesn’t require it